app/src/main/java/ru/vssemikoz/newsfeed/MainActivity.java

```diff
@@ -24,72 +25,98 @@
 import ru.vssemikoz.newsfeed.models.NewsApiResponseItem;
 import ru.vssemikoz.newsfeed.models.NewsApiResponse;
 import ru.vssemikoz.newsfeed.models.NewsItem;
+import ru.vssemikoz.newsfeed.storage.ApiStorage;
+import ru.vssemikoz.newsfeed.storage.NewsStorage;
 
 public class MainActivity extends AppCompatActivity implements PickCategoryDialog.OnCategorySelectedListener {
-    Category category = Category.ALL;
-    String KEY;
-    MainApplication mainApplication;
-    NewsItemDAO newsItemDAO;
-    Callback<NewsApiResponse> callbackNewsItemList;
-    List<NewsApiResponseItem> newsApiResponseItems = new ArrayList<>();
-    List<NewsItem> newsItemsFromDB = new ArrayList<>();
-    NewsFeedAdapter adapter;
-    RecyclerView recyclerView;
+    private String TAG = "MyLog";
```

Comment:
We usually putting a class name to filter logs. Like `MainActivity.class.getSimpleName()`.